README.md

```diff
@@ -21,87 +22,26 @@ Happa is a single page JavaScript application using React+Redux and runs in mode
 
 ## Getting started with development / demoing
 
-Happa has to talk to an API endpoint. It used to be possible to change a locally
-running happa's configuration to talk to any API server. This has since been
-restricted with CORS accept headers.
+Running Happa locally requires [NodeJS 12](https://nodejs.org/) and [Yarn](https://yarnpkg.com/).
 
-The only option now is to bring up the API suite locally, or demo happa running
-on one of our installations.
+You must have [opsctl](https://github.com/giantswarm/opsctl) installed and configured properly.
+opsctl is our internal tool for managing access to clusters and performing ops related tasks.
 
-### Bring up the API suite locally
+Bring up Happa and have it speak to the 'ginger' installation by using the following
+command:
 
-This requires Docker and `docker-compose`, and access to our private `api` repo.
-Currently only Giant Swarm employees are able to see that repo. We're considering
-open sourcing it.
-
-Start the `api` and dependencies first by going to the `api` [repo](https://github.com/giantswarm/api) and running
-the dockercompose file in the testing folder there:
-
-```nohighlight
-cd $GOPATH/src/github.com/giantswarm/api/testing
-make aws
-make up
 ```
-
-> Notice the `make aws` command in that example? There is also `make azure` and `make kvm`
-> to create a docker-compose file that sets up the API and other microservices as if
-> they were on these types of installations.
-
-As part of `make up`, `./fixtures.sh` will run and create the initial user and
-organization you can log in with.
-
-You should now be able to start happa's development server from within this
-(giantswarm/happa) repo like so:
-
-```nohighlight
-docker-compose up --build
+HAPPA_PROXY_INSTALLATION=ginger yarn start
```

Comment:
Do we want to mention this in a public repo?

Reply:
I think it's ok, It might be a bit confusing, but it's similar to how we linked and talked about the `api` repo, which is also not publicly accessible.

Reply:
anything specifically you think is a problem?

Reply:
I think mentioning `opsctl` is not a big deal, but not sure if providing the link to the repo is the right thing to do

Reply:
Hmm ok, I think it shouldn't be any different than how we also had the link to the API repo.

Is it from a security point of view or from a usability / understandability for non Giant Swarm staff point of view?

Reply:
Here are my 2 cents: Giant Swarm Employees should be already familiar with opsctl, so linking it shouldn't be neccesary. The code using it is alreay open source so mentioning it shouldn't be an issue